tools/validators/instance_validator/tests/fake_instances/GOOD/guid_format.yaml

```diff
@@ -20,18 +20,18 @@ US-SEA-BLDG1-GUID:
 CHWS-1-GUID:
   type: HVAC/CHWS_WDT
   code: CHWS-1
-  cloud_device_id: "foobar"
+  cloud_device_id: "2619178366980754"
```

Comment:
change cloud_device_id to something more generic like 1234567890123456

Reply:
Done.